knative-operator/pkg/common/health_dashboard.go

```diff
@@ -0,0 +1,86 @@
+package common
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	mfc "github.com/manifestival/controller-runtime-client"
+	mf "github.com/manifestival/manifestival"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var logh = Log.WithName("health dashboard")
+
+const ConfigManagedNamespace = "openshift-config-managed"
+
+func InstallHealthDashboard(api client.Client) error {
+	namespace, err := getOperatorNamespace()
+	if err != nil {
+		return err
+	}
+	instance, err := getServerlessOperatorDeployment(api, namespace)
+	if err != nil {
+		return err
+	}
+	err = api.Get(context.TODO(), client.ObjectKey{Name: ConfigManagedNamespace}, &corev1.Namespace{})
```

Comment:
The `openshift-config-managed` namespace verification should move up at L#22.

Reply:
fixed